app/models/comfy/cms/page.rb

```diff
@@ -97,7 +97,7 @@ def published_at
     elsif scheduled?
       scheduled_on
     elsif ever_been_published?
-      revisions.detect { |revision| revision.data[:event].include?('published') }.created_at
+      revisions.detect { |revision| Array(revision.data[:event]).include?('published') }.created_at
```

Comment:
It's maybe just me but I find this a little confusing to parse when reading. Would `revision.data[:event].try(:include?, 'published')` be more readable?